### DISCUSSION TITLE
Ререндер кластеров

### DISCUSSION THREAD

**Discussion by Wate835** [yandex-maps-unofficial/vue-yandex-maps#310] [Q&A (2.x)]
При зум кластеры ререндерятся, но как будто не полностью, у меня внутри содержатся попапы и при зуме, внутренние значения для отображения всплывашек виснут, при этом при клике значение в консоли я вижу что меняется. Я не уверен что это к вам вопрос, но я что то в тупике

**Comment by daniluk4000** [comment]
Скорее всего это из-за того, что там реактивности нет :(

Еще в настройки кластера можно передать функцию рендера, срабатывает каждый раз при ререндере. Можно попробовать ею триггерить какой-нибудь ключ для ререндера содержимого.

Звучит так себе, но я пока с ходу не знаю, что еще можно придумать
